.github/workflows/.ansible-test.yml

```diff
@@ -36,3 +36,43 @@ jobs:
 
       - name: Run sanity tests
         run: ansible-test sanity --docker -v --color --python ${{ matrix.python_version }}
+
+  units:
+    runs-on: ubuntu-latest
+    name: Units (Ⓐ${{ matrix.ansible }})
+    strategy:
+      # As soon as the first unit test fails, cancel the others to free up the CI queue
+      fail-fast: true
+      matrix:
+        python_version:
+          - 3.6
+          - 3.7
+        ansible:
+          - stable-2.10
```

Comment:
try to include ansible 2.9 and devel branch also, the same way we did in sanity tests